modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -83,8 +83,10 @@ platform:
     diskType: thin <7>
     network: VM_Network
     cluster: vsphere_cluster_name <8>
-    apiVIP: api_vip
-    ingressVIP: ingress_vip
+    apiVIPs: 
+      - api_vip
```

Comment:
does this account for multiple Vips?

Reply:
@cybertron - regarding Eldar's question, is this ok on the vSphere platform? I think dual stack VIPs isn't supported on vSphere yet so it's just a change of format for the moment?

Reply:
Correct. If/when we add dual stack support for other platforms we can/should update this, but in the meantime non-baremetal platforms can only take a single VIP anyway.